src/esm_runscripts/filedicts.py

```diff
@@ -65,8 +67,70 @@ def __init__(self, full_config, attrs_address):
     def cp(self) -> None:
         pass
 
-    def ln(self) -> None:
-        pass
+    def ln(
+        self, source_path: Union[AnyStr, os.PathLike], destination_path: str
```

Comment:
Again, do we want real paths here or are we just gonna give `"work"`, `"in_exp_tree"`, ... strings to point at the keys in the files dictionary to be used as paths. I don't see the need of using paths here, specially since this is not a general function but a method inside a class that defines the paths a properties in the initialization.

Reply:
This is resolved now